pyomo/repn/plugins/gams_writer.py

```diff
@@ -308,6 +308,27 @@ def split_long_line(line):
     return new_lines
 
 
+class GAMSSymbolMap(SymbolMap):
+    def __init__(self, var_labeler, var_list):
+        super().__init__(self.var_label)
+        self.var_labeler = var_labeler
+        self.var_list = var_list
+
+    def var_label(self, obj):
+        # if obj.is_fixed():
```

Comment:
Still needed?

Reply:
I don't know... and don't know the writer well enough to guess which breadcrumbs are safely deletable.  I was going to leave it in,